targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/TARGET_FRDM/PinNames.h

```diff
@@ -267,6 +241,38 @@ typedef enum {
     PullDefault = PullUp
 } PinMode;
 
+
+// LEDs
+#define LED_RED   PTB22
+#define LED_GREEN PTE26
+#define LED_BLUE  PTB21
+
+// Standardized LED names
+#define LED1 LED_RED
+#define LED2 LED_GREEN
+#define LED3 LED_BLUE
+
+
+//Push buttons
+#define SW2 PTC6
+#define SW3 PTA4
+
+// Standardized button names
+#define BUTTON1 SW2
+#define BUTTON2 SW3
+
+
+// SPI Pins for SD card.
+// Note: They are different from the Arduino Uno SPI pins
+// (ARDUINO_UNO_SPI_xxx) for general purpose uses.
+// By default, Mbed OS maps those alias to Arduino Uno pins, but
+// for backward compatibility we map them to the SD card SPI.
+#define SPI_MOSI PTE3
```

Comment:
This is also part of the migration, couldn't it stay in PinName enum and just add a comment or?

Reply:
The SPI pins need to be `#define`d: https://github.com/ARMmbed/mbed-os/blob/81aaef48b22373d61ec900b12f3aca5770688ace/hal/include/hal/PinNameAliases.h#L170-L173
If they were enums, the macro checks would not work.